src/main/java/com/maddyhome/idea/vim/ui/VimEmulationConfigurable.kt

```diff
@@ -328,6 +329,38 @@ internal class VimEmulationConfigurable : Configurable {
     }
   }
 
+  private class ToggleAllHandlersAction(
+    private val myModel: VimShortcutConflictsTable.Model,
+    private val myTable: VimShortcutConflictsTable,
+  ) : DumbAwareAction(
+    "Toggle all Handlers",
+    "Toggle all action handlers to use either IDE or Vim shortcuts",
+    AllIcons.Actions.ChangeView,
+  ) {
+    override fun update(e: AnActionEvent) {
+      e.presentation.isEnabled = true
+    }
+
+    override fun getActionUpdateThread(): ActionUpdateThread = ActionUpdateThread.EDT
```

Comment:
Let's switch to ActionUpdateThread.BGT

Reply:
Thanks for the feedback, will do another passover tonight with the above in mind.

I like the idea of not updating all handlers, seems "safer" for the user.

Reply:
I have added the above funtionality.